airflow/operators/python.py

```diff
@@ -243,13 +243,14 @@ def execute(self, context: Context) -> Any:
     def determine_kwargs(self, context: Mapping[str, Any]) -> Mapping[str, Any]:
         return KeywordParameters.determine(self.python_callable, self.op_args, context).unpacking()
 
-    def execute_callable(self) -> Any:
+    def execute_callable(self, dataset_events: DatasetEventAccessors) -> Any:
```

Comment:
I had to change the signature here (and in subclasses) so the callable can access the accessors. I _think_ this shouldn’t be public and thus not backward incompatible? Not exactly sure…

Reply:
Mhm, it is not exactly defined as "public" but at least we have also a custom subclass of python operator developed in our environment. So everybody having a subclass of this will see a breaking change?